shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -275,6 +275,24 @@ - (void)requestApplicationTermination:(id)sender
 
 #pragma mark -
 
+@implementation FlutterPasteboard
+
+- (NSInteger)clearContents {
+  return [[NSPasteboard generalPasteboard] clearContents];
```

Comment:
(Someday when we start using Swift in the engine, there's a cool trick Huan showed me where Swift lets you add a protocol to an existing class via extension—without implementing that protocol in the extension itself, like ObjC wants—which means this kind of direct passthrough is no longer needed, because you can make NSPasteboard itself be the default implementation of the protocol.)

Reply:
Oh nice. I played with making this a protocol but in Obj-C it ends up even more boilerplate than it already is as a subclass, what you're talking about sounds ideal.